<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Remove Background</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1921</width>
      <height>1051</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Assets/Remove_background.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="browse_lbl">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>230</y>
      <width>381</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: transparent; /* Default text color */
    font-size: 30px;
    font-weight: bold;
    border: 2px solid transparent; /* Transparent border by default */
    border-radius: 10px;
    background-color: transparent; /* Transparent background */
    qproperty-alignment: 'AlignCenter';
}

QLabel:hover {
    border-color: black; /* Visible border on hover */
    background-color: #CAF1DF; /* Light greenish-blue background */
    color: black;
}</string>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="remove_bg_lbl">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>390</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: transparent; /* Default text color */
    font-size: 30px;
    font-weight: bold;
    border: 2px solid transparent; /* Transparent border by default */
    border-radius: 10px;
    background-color: transparent; /* Transparent background */
    qproperty-alignment: 'AlignCenter';
}

QLabel:hover {
    border-color: black; /* Visible border on hover */
    background-color: #CAF1DF; /* Light greenish-blue background */
    color: black;
}</string>
    </property>
    <property name="text">
     <string>Remove BG</string>
    </property>
   </widget>
   <widget class="QLabel" name="view_lbl">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>540</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: transparent; /* Default text color */
    font-size: 30px;
    font-weight: bold;
    border: 2px solid transparent; /* Transparent border by default */
    border-radius: 10px;
    background-color: transparent; /* Transparent background */
    qproperty-alignment: 'AlignCenter';
}

QLabel:hover {
    border-color: black; /* Visible border on hover */
    background-color: #CAF1DF; /* Light greenish-blue background */
    color: black;
}</string>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QLabel" name="input_image_lbl">
    <property name="geometry">
     <rect>
      <x>536</x>
      <y>232</y>
      <width>601</width>
      <height>521</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="output_lbl">
    <property name="geometry">
     <rect>
      <x>1046</x>
      <y>622</y>
      <width>391</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="reset_lbl">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>710</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Assets/reset_button.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="path_lbl">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>320</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="edit_mode_lbl">
    <property name="geometry">
     <rect>
      <x>1630</x>
      <y>830</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>edit</string>
    </property>
   </widget>
   <widget class="QLabel" name="editor_lbl">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>1061</width>
      <height>701</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
